media-sound/jack2/jack2-1.9.11_rc1.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="threads(+)"
+[[ "${PV}" = "2.9999" ]] && inherit git-r3
```

Comment:
Do you mean that if I inherit git-r3, that it's src_unpack will be used? If so, how can I know this? Is this caused by [`EXPORT_FUNCTIONS`](https://github.com/gentoo/gentoo/blob/master/eclass/git-r3.eclass#L20)?